tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -56,30 +32,18 @@ import scala.concurrent.Future
 import scala.meta.Input
 import scala.meta.internal.io.FileIO
 import scala.meta.internal.io.PathIO
-import scala.meta.internal.metals.Buffers
-import scala.meta.internal.metals.Debug
-import scala.meta.internal.metals.DidFocusResult
-import scala.meta.internal.metals.WindowStateDidChangeParams
-import scala.meta.internal.metals.Directories
 import scala.meta.internal.metals.MetalsEnrichments._
-import scala.meta.internal.metals.MetalsLanguageServer
-import scala.meta.internal.metals.MetalsServerConfig
 import scala.meta.internal.metals.PositionSyntax._
-import scala.meta.internal.metals.ProgressTicks
-import scala.meta.internal.metals.Time
-import scala.meta.internal.metals.UserConfiguration
+import scala.meta.internal.metals._
```

Comment:
Let's not use wildcard imports, they often produce unexpected results and might make compilation slover